platform/core/src/services/Throttle.js

```diff
@@ -56,12 +61,14 @@ define(
              * @memberof platform/core.Throttle#
              */
             return function (fn, delay, apply) {
-                var activeTimeout;
+                var promise, // Promise for the result of throttled function
```

Comment:
Lots of unnecessary comments here-- duplicating what's already stated by the code.